daemons/attrd/attrd_commands.c

```diff
@@ -1023,6 +1052,20 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
     }
 }
 
+void
+attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
+{
+    if (xml_has_children(xml)) {
+        for (xmlNode *child = first_named_child(xml, XML_ATTR_OP); child != NULL;
+             child = crm_next_same_xml(child)) {
+            attrd_peer_update_one(peer, child, filter);
```

Comment:
Is it guaranteed that the children always have PCMK__XA_ATTR_NODE_NAME set?

Reply:
No, I don't think so.

Reply:
Added.